pkg/api/handlers/compat/containers_stats.go

```diff
@@ -175,11 +191,18 @@ func StatsContainer(w http.ResponseWriter, r *http.Request) {
 			Networks: net,
 		}
 
-		utils.WriteJSON(w, http.StatusOK, s)
+		if err := coder.Encode(s); err != nil {
+			utils.InternalServerError(w, err)
```

Comment:
```suggestion
			logrus.Warnf(...)
```
You already sent 200 to client  you cannot send a 500 here.

Reply:
Thanks for catching! I changed all other occurrences as well (but used Errorf).